### ISSUE TITLE
mips: Do not set general exception vector correctly

### ISSUE THREAD

**Issue by 123serge123** [embox/embox#3483]
In *exc_init.c*  mips_first_exception_handler is copied to fixed address KSEG0+0x180 but never check/set c0 EBase register.
At least need to call **mips_write_c0_ebase(KSEG0)**

**Comment by 123serge123** [comment]
Currently I use this simple patch:
```diff --git a/src/arch/mips/kernel/exc_init.c b/src/arch/mips/kernel/exc_init.c
index 1f3518c..1a19f6a 100644
--- a/src/arch/mips/kernel/exc_init.c
+++ b/src/arch/mips/kernel/exc_init.c
@@ -24,6 +24,9 @@ void mips_exception_init(void) {
 
 	flush_cache((unsigned long)(KSEG0 + 0x180), 0x80);
 
+	/* explicitly set ebase */
+	mips_write_c0_ebase(KSEG0);
+
 	execution_hazard_barrier();
 
 	/* Configure status register */
```

**Comment by anton-bondarev** [comment]
> Currently I use this simple patch:
> 
> ```diff
> index 1f3518c..1a19f6a 100644
> --- a/src/arch/mips/kernel/exc_init.c
> +++ b/src/arch/mips/kernel/exc_init.c
> @@ -24,6 +24,9 @@ void mips_exception_init(void) {
>  
>  	flush_cache((unsigned long)(KSEG0 + 0x180), 0x80);
>  
> +	/* explicitly set ebase */
> +	mips_write_c0_ebase(KSEG0);
> +
>  	execution_hazard_barrier();
>  
>  	/* Configure status register */
> ```

Great!
Could you please create a PR with these changes?

**Comment by 123serge123** [comment]
Never did it before but I'll try :)

**Comment by 123serge123** [comment]
@AlekseyZhmulin 
Formally correct but for prev mips isa revisions EBase register may be optional.

**Comment by 123serge123** [comment]
Fixed in commit (22685b9fcf3cb457719277e2b4963df7d7af1460)
